include/libssh2.h

```diff
@@ -716,6 +716,11 @@ libssh2_channel_direct_tcpip_ex(LIBSSH2_SESSION *session, const char *host,
 #define libssh2_channel_direct_tcpip(session, host, port) \
   libssh2_channel_direct_tcpip_ex((session), (host), (port), "127.0.0.1", 22)
 
+LIBSSH2_API LIBSSH2_CHANNEL *
+libssh2_channel_direct_streamlocal_ex(LIBSSH2_SESSION * session,
+                                const char *socket_path,
+                                const char *shost, int sport);
```

Comment:
I'd be nice to align these.
```suggestion
                                      const char *socket_path,
                                      const char *shost, int sport);
```